modules/selenium/src/main/java/org/testcontainers/containers/SeleniumUtils.java

```diff
@@ -0,0 +1,70 @@
+package org.testcontainers.containers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Enumeration;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.jar.Attributes;
+import java.util.jar.Manifest;
+
+/**
+ * Utility methods for Selenium.
+ */
+public class SeleniumUtils {
```

Comment:
this class could be also made final according to best practices for utility classes